### PR TITLE
Note the existence of a `execroot/` directory

### PR DESCRIPTION
**Pull request by alexmv**
To be true to the layout on disk, one should note the literal directory named `execroot` under the outputBase.

Also fix the indentation on `<workspace-name>/` due to the HTML escapes.

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by michelleirvine** [comment]
@alexmv is there a useful description for `execroot/` we can add on the right? If not we can leave as-is (some other directories also don't have descriptions).

**Comment by laszlocsomor** [comment]
I think a good summary of `execroot/` would be, it's the working directory of all actions (it's the "execution root"), except for special cases (sandboxing, remote execution).

**Comment by laszlocsomor** [comment]
And another thing, "execroot" is more of a concept than an actual directory on the filesystem.

In case of sandboxing for example, we create an execroot-looking directory and that's where the action runs. With remote execution, the same is true. Where exactly those directories are created, on what kind of filesystem -- that's an implementation detail, intentionally hidden from the action.

All the action needs to know is that it runs in some directory, which we call the execroot, and it can access its inputs and outputs relative to that directory.

**Comment by michelleirvine** [comment]
Okay - this is a bit long, but how about:

The working directory for all actions. For special cases such as sandboxing and remote execution, the actions run in a directory that mimics `execroot` with implementation details, such as where the directories are created, intentionally hidden from the action. All actions can access its inputs and outputs relative to the `execroot` directory.

**Comment by michelleirvine** [comment]
I've made that change in a commit on this pull request. @alexmv does this look good or do you want any edits?

**Comment by alexmv** [comment]
Wording on the `execroot` looks good, but NAK on the indentation changes.  My indentation changes on the lines with `&lt;`/`&gt;` were intentional, and are necessary to have the `<==` bits line up in the rendered markdown form.

Contrast the alignment in https://github.com/alexmv/bazel/blob/a02a9090389f1d1b552af09dcc59f6da1fd3464f/site/docs/output_directories.md#bazel-internals-directory-layout (after your change) with the alignment in https://github.com/alexmv/bazel/blob/7939487ea365f679eec96a67686152a9ab077d92/site/docs/output_directories.md#bazel-internals-directory-layout (after my change)

File: site/docs/output_directories.md [hunk 1]
```diff
@@ -90,36 +90,42 @@ The directories are laid out as follows:
       server/                             <== The Bazel server puts all server-related files (such as socket
                                               file, logs, etc) here.
         jvm.out                           <== The debugging output for the server.
-      &lt;workspace-name&gt;/             <== Working tree for the Bazel build & root of symlink forest: execRoot
-        _bin/                             <== Helper tools are linked from or copied to here.
-
-        bazel-out/                        <== All actual output of the build is under here: outputPath
-          local_linux-fastbuild/          <== one subdirectory per unique target BuildConfiguration instance;
+      execroot/                           <== The working directory for all actions. For special cases such
```
Comment (laszlocsomor) [inline site/docs/output_directories.md]: > The working directory for all actions. For special cases such as sandboxing and remote execution, the actions run in a directory that mimics execroot.

Please end the sentence here and begin a new one at "Implementation details". It was difficult for me to parse the sentence in its current form.

**Comment by michelleirvine** [comment]
Fixed indenting and split the long sentence.

**Comment by laszlocsomor** [comment]
I just noticed that this PR was never upstreamed. Sorry! Let me try to fix that.

**Pull request closed without merging**